package.json

```diff
@@ -83,7 +83,7 @@
     "gitbook-cli": "^2.3.2",
     "husky": "^0.14.3",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^21.2.1",
+    "jest": "^22.2.1",
```

Comment:
What an odd coincidence, the latest version of Jest happens to have the same minor and patch version integer. 🍨